.github/workflows/stale.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '41 10 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: "This issue hasn't been updated for a while, marking as stale, please respond within the next 7 days to remove this label"
+        stale-pr-message: "This PR hasn't been updated for a while, marking as stale"
+        stale-issue-label: 'stale'
+        stale-pr-label: 'stale'
+        # mark issues and PR's as stale after this many days
+        days-before-stale: 60
```

Comment:
I think 30 days is enough for this repo, we are active enough as maintainers and we don't want to many open issues.